cmd/yggd/grpc.go

```diff
@@ -113,6 +116,33 @@ func (d *dispatcher) Send(ctx context.Context, r *pb.Data) (*pb.Receipt, error)
 	return &pb.Receipt{}, nil
 }
 
+func (d *dispatcher) sendEvents() {
+	for data := range d.eventsQ {
+		d.RLock()
+		w, prs := d.workers[data.Directive]
```

Comment:
When registering the worker, the key is "device", but here I'm not sure if we should range over *ALL* workers and send it the information.

Reply:
I think we need to